README.md

```diff
@@ -315,6 +315,8 @@ $ glab config set editor vim --host gitlab.example.org
 
   GITLAB_URI or GITLAB_HOST: specify the url of the gitlab server if self hosted (eg: https://gitlab.example.com). Default is https://gitlab.com.
 
+  GITLAB_API_HOST: specify the host where the endpoint is found: defaults to the hostname
```

Comment:
```suggestion
  GITLAB_API_HOST: specify the host where the API endpoint is found. Useful when there are separate [sub]domains or hosts for git and the API endpoint: defaults to the hostname found in the git URL
```

Same thing should be at 
https://github.com/profclems/glab/blob/222c0c51dd4288189fba8f50b1dd6118029ffe22/commands/root.go#L37-L60

Reply:
why is the documentation duplicated and slightly different?

in README
```
GITLAB_URI or GITLAB_HOST: specify the url of the gitlab server if self hosted (eg: https://gitlab.example.com). Default is https://gitlab.com.
```

in root.go
```
GITLAB_HOST or GL_HOST: specify the url of the gitlab server if self hosted (eg: https://gitlab.example.com). Default is https://gitlab.com.
```

Reply:
I assume some ENV Variables are kind of `old` and should be deprecated to clean out the logic and documentation.
What about introducing a way to print out a warning when the usage of old vars is detected? (in a separate PR)

Reply:
Can we generate one out the other? Or just point to the other documentation? Or include it, I know in ASCII Doc this is possible. The current state feels not right.

Reply:
Sorry I couldn't get back to you earlier. Had to take some break since it's Sunday

Reply:
I know the current state is a bit off. I had actually thought about refactoring but the readme uses markdown and it's not possible to centralise it. Maybe we would have to remove it from the docs so the users rely on the `help` command (But what about those using it in their CI environment only)

Reply:
> What about introducing a way to print out a warning when the usage of old vars is detected? (in a separate PR)

I agree with this... Should be a separate PR